pkg/common/genericSet.go

```diff
@@ -90,3 +90,12 @@ func MapValues[K comparable, V any](m map[K]V) []V {
 	}
 	return vals
 }
+func MapKeys[K comparable, V any](m map[K]V) []K {
```

Comment:
`GenericSet.AsList()` should use this function

Reply:
fixed